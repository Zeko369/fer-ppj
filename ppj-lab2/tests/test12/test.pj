za a od 0 do 1
az
