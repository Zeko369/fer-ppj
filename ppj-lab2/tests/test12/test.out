<program>
 <lista_naredbi>
  <naredba>
   <za_petlja>
    KR_ZA 1 za
    IDN 1 a
    KR_OD 1 od
    <E>
     <T>
      <P>
       BROJ 1 0
      <T_lista>
       $
     <E_lista>
      $
    KR_DO 1 do
    <E>
     <T>
      <P>
       BROJ 1 1
      <T_lista>
       $
     <E_lista>
      $
    <lista_naredbi>
     $
    KR_AZ 2 az
  <lista_naredbi>
   $
